introduction.md

```diff
@@ -0,0 +1 @@
+Hi there!  My name's David Reghay and I would like to study at your fine dojo!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.